lib/globus/client/endpoint.rb

```diff
@@ -80,6 +80,7 @@ def call_mkdir(path)
         response
       end
 
+      # Makes the API call to Globus to set permissions
```

Comment:
Typo.

Reply:
What is the typo?

Reply:
I don't see a typo either. I'll leave to @justinlittman to merge if he concurs